lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -1809,14 +2899,18 @@ DomHTMLIFrameElement createDomHTMLIFrameElement() =>
 class DomMessagePort extends DomEventTarget {}
 
 extension DomMessagePortExtension on DomMessagePort {
-  void postMessage(Object? message) => js_util.callMethod(this, 'postMessage',
-      <Object?>[if (message != null) js_util.jsify(message)]);
-  external void start();
+  @JS('postMessage')
+  external JSVoid _postMessage(JSAny? message);
+  void postMessage(Object? message) => _postMessage(message.toJSAnyShallow);
+
+  external JSVoid start();
 }
 
-@JS()
+@JS('MesageChannel')
```

Comment:
This is misspelled, which I think is causing the unit test failures

Reply:
Good catch!